demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -98,8 +98,7 @@ def check_deprecated(self, file_path):
         if file_path.endswith('.yml'):
             yml_dict = get_yaml(file_path)
             if ('deprecated' in yml_dict and yml_dict['deprecated'] is True) or \
-                    (file_name.startswith('playbook') and 'hidden' in yml_dict and
-                     yml_dict['hidden'] is True):
+                    (file_name.startswith('playbook') and 'hidden' in yml_dict and yml_dict['hidden'] is True):
```

Comment:
In a conversation with Roy he told me to change here something he had written before because it did not look good, and after I changed there were tests that did not pass, so I changed it back.

Reply:
In a conversation with @roysagi  he told me to change here something he had written before because it did not look good, and after I changed there were tests that did not pass, so I changed it back.

Reply:
@ChanochShayner , @guykeller - As I told Chanoch the file name does not have to start with "playbook" to be a playbook, which makes this code incorrect - therefore in need of a fix.
@ChanochShayner - please revert to the changes we made and check which unit tests failed - we should fix the unit tests and not ignore this change.